thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js

```diff
@@ -0,0 +1,65 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  entities: null, // {}
+
+  selectedUrns: null, // Set
+
+  timeseries: null, // {}
+
+  series: Ember.computed(
+    'entities',
+    'timeseries',
+    'selectedUrns',
+    function () {
+      const entities = this.get('entities');
+      const timeseries = this.get('timeseries');
+      const selectedUrns = this.get('selectedUrns');
```

Comment:
use `this.getProperties` to consolidate these 3 lines https://emberjs.com/api/ember/2.15/classes/Ember.Object/methods/getProperties?anchor=getProperties&show=inherited%2Cprotected%2Cprivate%2Cdeprecated

Reply:
fixed